src/unity/python/doc/source/turicreate.toolkits.one_shot_object_detector.rst

```diff
@@ -0,0 +1,24 @@
+:mod:`one_shot_object_detector`
+==========================
+
+.. automodule:: turicreate.toolkits.one_shot_object_detector
+
+.. currentmodule:: turicreate.one_shot_object_detector
+
+
+one shot object detector
+---------------
+.. autosummary::
+  :toctree: generated/
+  :nosignatures:
+
+  create
+  OneShotObjectDetector
+
+utilities
+---------
+.. autosummary::
+  :toctree: generated/
+  :nosignatures:
+
+  util.preview_augmented_images
```

Comment:
Do we also want to add `util.check_one_shot_input`?

Reply:
We should ask @shantanuchhabra. It had nothing in comments so I didn't add it yet.

Reply:
I don't think we should add that. That;s probably an underscore error.

Reply:
@TobyRoseman The _ error will be fixed in #1986